graph/db/notifications.go

```diff
@@ -127,23 +127,22 @@ func (b *Builder) notifyTopologyChange(topologyDiff *TopologyChange) {
 			len(topologyDiff.ChannelEdgeUpdates),
 			len(topologyDiff.ClosedChannels))
 
-		go func(c *topologyClient) {
-			defer c.wg.Done()
+		go func(t *topologyClient) {
```

Comment:
ok i didn't realize we were firing goroutine inside goroutine inside yet anther goroutine😂

Reply:
yeah it's all very gross 😂

Reply:
This should be fine though.

Reply:
:this-is-fine: